### PR TITLE
 fixing editig cohort

### PR DESCRIPTION
**Pull request by HemaSAli**

### CODE REVIEW

File: public/css/projectWebsite.css [hunk 1]
```diff
@@ -79,3 +79,144 @@ h5 {
   margin-left: 2px;
   list-style: decimal;
 }
+
+
+.imageClient{
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: `image-client`

File: public/css/projectWebsite.css [hunk 2]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+  width: 100%;
+}
+
+h4 {
+  font-weight: bold;
+  margin: 0;
+}
+.w3-display-bottommiddle2 {
```
Comment (NouraldinS) [inline public/css/projectWebsite.css:84]: `w3-display-bottm-middle-2`

File: public/css/projectWebsite.css [hunk 3]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+  position: absolute;
+  left: 50%;
+  bottom: 173px;
+  transform: translate(-50%, 0%);
+  -ms-transform: translate(-50%, 0%);
+}
+/* body */
+
+.section2 {
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: `section-2`

File: public/css/projectWebsite.css [hunk 4]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+  padding: 2%;
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-gap: 10px;
+  width: 75%;
+  margin: 0 auto;
+}
+.textmain {
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: `text-main`

File: public/css/projectWebsite.css [hunk 5]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+  overflow: hidden;
+  /* height: 150px; */
+  font-weight: bold;
+  font-size: 15px;
+  font-family: Roboto;
+  line-height: 28px;
+  letter-spacing: 1px;
+  color: #757575;
+}
+.client_project{
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: `client-project`

File: public/css/projectWebsite.css [hunk 6]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+.client_project>div{
+  text-align: center;
+  -webkit-box-shadow: 0px 0px 23px -5px rgba(0,0,0,0.75);
+-moz-box-shadow: 0px 0px 23px -5px rgba(0,0,0,0.75);
+box-shadow: 0px 0px 23px -5px rgba(0,0,0,0.75);
+border-radius: 5px;
+color: #757575;
+
+}
+.projectImg{
```
Comment (NouraldinS) [inline public/css/projectWebsite.css:141]: `project-img`

File: public/css/projectWebsite.css [hunk 7]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+  border-radius: 5px;
+
+  max-width: 100%;
+}
+
+.informationUl{
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: `information-ui`

File: public/css/projectWebsite.css [hunk 8]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+
+  margin-left: 7%;
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: Do not use relaitve margins.

File: public/css/projectWebsite.css [hunk 9]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+  font-weight: bold;
+  color: #757575;
+  margin-bottom: 3%;
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: Do not use relative margins.

File: public/css/projectWebsite.css [hunk 10]
```diff
@@ -79,3 +79,144 @@ h5 {
...
+.informationUl>li{
+    margin-bottom: 1%;
+}
+
+.sutdentsUl{
+    margin-left: 5%;
+    list-style: decimal; 
+}
+/* footer */
+.img_footer {
```
Comment (NouraldinS) [inline public/css/projectWebsite.css]: `img-footer`

File: src/controllers/editCohort.js [hunk 11]
```diff
@@ -18,7 +18,9 @@ exports.getCohortData = (req, response, next) => {
 exports.editCohort = (req, response, next) => {
   const data = req.body;
   editCohorts(data)
+
```
Comment (NouraldinS) [inline src/controllers/editCohort.js]: Remove extra newlines.

**Review by NouraldinS — CHANGES_REQUESTED** [review]

**Review by NouraldinS — APPROVED** [review]

**Review by ali-7 — APPROVED** [review]

**Pull request merged**